worlds/factorio/Client.py

```diff
@@ -120,7 +120,7 @@ def on_print_json(self, args: dict):
 
     @property
     def savegame_name(self) -> str:
-        return f"AP_{self.seed_name}_{self.auth}_Save.zip"
+        return get_file_safe_name(f"AP_{self.seed_name}_{self.auth}")+"_Save.zip"
```

Comment:
Why is the constant suffix "_Save.zip" excluded from the method call, but the constant prefix "AP_" isn't?
I don't think either approach is particularly critical, but the inconsistency strikes me as strange